about/for-proposals.rst

```diff
@@ -112,14 +144,34 @@ this grant.  During this period, one of the Aalto research software engineers jo
 project as a researcher, equal to all other project employees.
 
 
+Other computing and IT solutions
+================================
```

Comment:
We shouldn't make this the same as the page-level heading, otherwise there is no title... 
```suggestion
-------------------------------
```